modules/sdk-coin-atom/src/lib/keyPair.ts

```diff
@@ -0,0 +1,74 @@
+import { randomBytes } from 'crypto';
+import { bip32 } from '@bitgo/utxo-lib';
+import { pubkeyToAddress } from '@cosmjs/amino';
+import { toBase64 } from '@cosmjs/encoding';
+import {
+  DefaultKeys,
+  isPrivateKey,
+  isPublicKey,
+  isSeed,
+  KeyPairOptions,
+  Secp256k1ExtendedKeyPair,
+} from '@bitgo/sdk-core';
+
+const DEFAULT_SEED_SIZE_BYTES = 16;
```

Comment:
Is this common across chains (does seem like it but confirming) ? If yes, do we need to put it somewhere in the `Secp256k1ExtendedKeyPair` class or any other appropriate file ?

Reply:
You are correct, it seems most commonly 16, but some chains have it at 64.
For our purposes, I will set it in a coin constants file.